### DISCUSSION TITLE
Welcome to newbark-unity Discussions!

### DISCUSSION THREAD

**Discussion by itsjavi** [capsulemonsters/newbark#33] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this project.

**Comment by laithnurie** [comment]
hey @itsjavi , just wanted to say hi and actually thank you for doing this. I am learning unity and I wanted to start with a pokemon style game as a playground to learn from.

Thanks again,
Laith

**Reply by itsjavi** [reply]
I am happy that it is useful for someone. It has a lot of room for improvement, but it helped me to start learning Unity as well.
